installation/resources/crds/ory/rules.crd.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
If we are removing this space here we should remove it in oauth2clients.crd too

Reply:
Thanks for reminder, will do later on